client/extensions/woocommerce/state/ui/orders/actions.js

```diff
@@ -2,17 +2,17 @@
  * Internal dependencies
  */
 import { getSelectedSiteId } from 'state/ui/selectors';
-import { WOOCOMMERCE_UI_ORDERS_SET_PAGE } from 'woocommerce/state/action-types';
+import { WOOCOMMERCE_UI_ORDERS_SET_QUERY } from 'woocommerce/state/action-types';
 
-export const setCurrentPage = ( siteId, page ) => ( dispatch, getState ) => {
+export const setCurrentQuery = ( siteId, query ) => ( dispatch, getState ) => {
```

Comment:
Would you consider renaming `setCurrentQuery` to something like `setOrdersCurrentQuery` so that, down the road, if we have non-order query actions dispatched from components that dispatch order-related query actions, we can tell them apart readily?